src/syscalls/interfaces/mod.rs

```diff
@@ -49,6 +49,9 @@ pub trait SyscallInterface: Send + Sync {
 	}
 
 	fn shutdown(&self, error_code: i32) -> ! {
+		// This is a stable message used for detecting exit codes for different hypervisors.
+		println!("exit status {error_code}");
```

Comment:
One could debate, whether `exit_status={error_code}` might be easier to parse. But I'm fine with it as is.